docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -17,6 +17,94 @@ import TimelineDot from '@mui/lab/TimelineDot';
 import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
 
 const entries = [
+  {
+    title: 'MUI X v7.19',
+    description: 'A roundup of all new features since v7.15.0.',
+    date: new Date(2024, 9, 4),
+    url: 'https://github.com/mui/mui-x/releases/tag/v7.19.0',
+    highlightList: [
+      {
+        title: 'Data Grid – Row spanning',
+        url: '/x/react-data-grid/row-spanning/',
+      },
+      {
+        title: 'Data Grid – Automatic parents and children selection',
+        url: '/x/react-data-grid/tree-data/#automatic-parents-and-children-selection',
```

Comment:
MIght be better to directly link the main docs.
```suggestion
        url: '/x/react-data-grid/row-grouping/#automatic-parents-and-children-selection',
```